packages/react-component-library/src/components/TextInputE/TextInputE.test.tsx

```diff
@@ -41,7 +41,7 @@ describe('TextInputE', () => {
     it('should apply the `$hasContent` style rule to label', () => {
       expect(wrapper.getByTestId('text-input-label')).toHaveStyleRule(
         'transform',
-        'translate(11px,8px) scale(0.75)'
+        'translate(11px,6px) scale(0.75)'
```

Comment:
Missing whitespace after comma

Reply:
This one's a test – it comes through like that (from jsdom or whatever is normalising it...)